### PR TITLE
#143 Window Design Update + UI improvements

### PR DESCRIPTION
**Pull request by bejow**
## Related Ticket/Issue
#143 

## Description
New design and UI improvements for the Window Component

### CODE REVIEW

File: src/functional_components/Window/Window.module.css [hunk 1]
```diff
@@ -9,32 +9,32 @@
...
+
     top: 0;
     right: 0;
     bottom: 0;
     left: 0;
-    background-color: black;
-    opacity: 0.5;
+    opacity: 0.3;
+    background: #fff;
+    backdrop-filter: blur(5px);
```
Comment (ludwigschubi) [inline src/functional_components/Window/Window.module.css]: This doesn't work for my mac

File: src/stateful_components/Drive/Drive.js [hunk 2]
```diff
@@ -485,28 +486,22 @@ class Drive extends React.Component {
                     onSubmit={(selectedItems) => {
                         deleteItems(selectedItems, currentPath);
                     }}
-                    className={
-                        isConsentWindowVisible ? styles.visible : styles.hidden
-                    }
                     onClose={this.closeConsentWindow}
+                    visible={isConsentWindowVisible}
```
Comment (ludwigschubi) [inline src/stateful_components/Drive/Drive.js]: This will create Merge conflicts later 4 sure

Comment (ludwigschubi) [reply src/stateful_components/Drive/Drive.js]: So we need to add this to the `<Windows/>` component later

File: src/functional_components/Window/Window.module.css (hunk 1 above)
Comment (bejow) [reply src/functional_components/Window/Window.module.css]: Yes I investigated in that topic but I didn't found a solution that works on every browser. What I found was how to apply blur to a background image but not as an Overlay. I would leave it for now or do you have an idea how to achieve it?

**Comment by bejow** [comment]
> Could you replace the buttons in the windows with ActionButtons? Other than that it looks good

I did so but I'm not sure about the design. I added white and red to ActionBUttons colors but can you look at the color values and change them if you're not happy with them?

File: src/functional_components/Window/Window.module.css (hunk 1 above)
Comment (ludwigschubi) [reply src/functional_components/Window/Window.module.css]: alright, then let's make the border of the window #000 and 1px thick

Comment (ludwigschubi) [reply src/functional_components/Window/Window.module.css]: at least for now

**Pull request merged**